### PR TITLE
Replace ros test

### PR DESCRIPTION
**Pull request by DavidMansolino**
**Description**

This PR removes the ros test on travis (as it is often making the nightly fail) and add it to Github Actions (which doesn't block the nightly if failed and use a more recent version of ROS1).

It also introduced a new tag `test ros` which allows to trigger the ros test only (or in addition to other tests).

### CODE REVIEW

File: tests/ros.sh [hunk 1]
```diff
@@ -2,13 +2,14 @@
 
 echo @@@ Compile ros webots_ros package
 source /opt/ros/$ROS_DISTRO/setup.bash
-[ -d $WEBOTS_HOME/webots_catkin_ws ] && rm -r $WEBOTS_HOME/webots_catkin_ws
-mkdir -p $WEBOTS_HOME/webots_catkin_ws/src
-cd $WEBOTS_HOME/webots_catkin_ws/src
+BASEDIR=$(dirname $(realpath $0))/..
```
Comment (omichel) [inline tests/ros.sh:5]: We should maybe explain in a comment why BASEDIR is defined and why we don't use WEBOTS_HOME instead.

Comment (DavidMansolino) [inline tests/ros.sh:5]:
```suggestion
# BASEDIR might differ from WEBOTS if we are not using Webots compiled from this repository directory (e.f. in CI).
BASEDIR=$(dirname $(realpath $0))/..
```

@omichel would this be clear enough?

Comment (omichel) [reply tests/ros.sh:5]: Yes, thank you.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**